pyrefinebio/compendia.py

```diff
@@ -95,6 +98,29 @@ def download(self, path, prompt=True):
 
             prompt (bool): if true, will prompt before downloading files bigger than 1GB
         """
-        self.computed_file.download(path, prompt)
+        if not self.computed_file.download_url:
+            raise DownloadError("Compendia", "Download url not found - did you set up and activate a Token?")
+
+        full_path = expand_path(path, "compendia-" + str(self.id) + ".zip")
```

Comment:
Shouldn't we check to see if the path is a file or a directory before appending the filename to it?

Reply:
`expand_path` should [already do this](https://github.com/AlexsLemonade/refinebio-py/pull/39/files#diff-fbbdc042b88d5d41b2743515f83a29f5c1e8e794dbc321196453a87f2722143eR24-R25).

It'll only append the filename part if `path` is a dir.

Reply:
Oh beautiful!